tests/testOpStreamingH5N5SequenceReaderS.py

```diff
@@ -17,6 +16,9 @@ class TestOpStreamingH5N5SequenceReaderS(unittest.TestCase):
 
     def setup_method(self, method):
         self.graph = Graph()
+        self.tempdir = tempfile.TemporaryDirectory()
```

Comment:
We could use pytest's [`tmp_path` fixture](https://docs.pytest.org/en/latest/tmpdir.html#the-tmp-path-fixture) and get a temp dir for free. Although I'm not sure how this interacts with the class-based unittest tests...